alternate-installation-options/index.md

```diff
@@ -8,20 +8,23 @@ layout: default
 {:.cryostat-heading-1}
 Cryostat {{ site.data.versions.cryostat.version }}
 
+* auto-gen TOC:
+{:toc}
+
 ## [Alternate Installation Options](#alternate-installation-options)
 
 The recommended method to install the Cryostat Operator is via OperatorHub on the [Get Started](/get-started) page. If needed, see below for alternative methods to install the Cryostat Operator. All of the installation options require [cert-manager](/get-started#install-cert-manager) as a prerequisite.
 
 Once the operator is installed, proceed to the [Setup](/get-started#setup) section of the Get Started page.
 
-### Install with kubectl
+### [Install with kubectl](#install-with-kubectl)
 ```
 $ kubectl create namespace cryostat-operator-system
 $ kubectl apply -k 'github.com/cryostatio/cryostat-operator//config/default?ref=v{{ site.data.versions.cryostat.version }}'
```

Comment:
This link seems inaccessible to me: `github.com/cryostatio/cryostat-operator//config/default?ref=v2.3.0`?

Reply:
Hmm, yea, this procedure doesn't work anymore and hasn't for quite some time since we made changes to the YAML that was originally there I guess, and now it has `kustomize` and things to generate it. @ebaron any ideas for what to replace this with?

Reply:
This is probably because the `v2.3.0` tag doesn't exist yet. I'm not sure how much value there is in this installation method though. It's very easy to set up, but we should probably direct users to OperatorHub.io for upstream.

Reply:
No, that would only be useful in limited circumstances like pre-releases. I think it's worth trying to simplify things by removing both.